grammars/prql-lezer/src/prql.grammar

```diff
@@ -38,7 +38,7 @@ CallExpression { Identifier ArgList { (NamedArg | Assign | test)+ } }
 
 NamedArg { identPart ":" expression }
 Assign { identPart "=" expression }
-AssignCall { identPart "=" expression }
+AssignTupleItem { PropertyName { identPart } "=" expression }
```

Comment:
I definitely think that `AssignTupleItem` & `Assign` should be formatted the same. So we could name the LHS something like `Alias` or `Assignee` or `LHS` (but not property, we don't have properties) — if we do we should update them both...

Reply:
I agree that `Assign` should have the `identPart` wrapped in a public token so it is exposed as a child token of `Assign` because it increases the semantic richness of the grammar.

`Assign` is just used in the context of `CallExpression` so it could be renamed `Parameter` and have the child token `ParameterName`.

The grammar rules for the `Assign` token and the `AssignTupleItem` token would be equal. I don't know if it makes sense to have them as separate tokens or merge them into token that we re-use. `Parameter` and `ParameterName` would be unsuitable names for tokens inside a `TokenExpression` though, so probably keep them as separate tokens.

Reply:
OK! Happy to have a look at the next version...